### PR TITLE
dev-lang/ispc: Update live ebuild

### PR DESCRIPTION
**Pull request by DarkDefender**
This updates the ebuild to use the "use only upstream llvm features" flag (`DISPC_NO_DUMPS`) and also lifts the llvm 10 requirement as it now compiles and passes the automated tests with llvm 11 and 12.

### CODE REVIEW

File: dev-lang/ispc/ispc-9999.ebuild [hunk 1]
```diff
@@ -36,10 +36,12 @@ BDEPEND="
 
 PATCHES=(
 	"${FILESDIR}/${PN}-1.13.0-cmake-gentoo-release.patch"
-	"${FILESDIR}/${PN}-1.14.0-llvm-10.patch"
-	"${FILESDIR}/${PN}-1.13.0-werror.patch"
+	"${FILESDIR}/${PN}-9999-llvm.patch"
+	"${FILESDIR}/${PN}-9999-werror.patch"
 )
 
+CMAKE_BUILD_TYPE="Release"
```
Comment (telans) [inline dev-lang/ispc/ispc-9999.ebuild:43]: Why is this needed?

Comment (DarkDefender) [reply dev-lang/ispc/ispc-9999.ebuild:43]: A check has been added so trying to use anything but the standard build types will make the cmake scripts error out.

File: dev-lang/ispc/ispc-9999.ebuild [hunk 2]
```diff
@@ -7,7 +7,7 @@ PYTHON_COMPAT=( python3_{7,8,9} )
 
 inherit cmake toolchain-funcs python-any-r1 llvm
 
-LLVM_MAX_SLOT=10
+LLVM_MAX_SLOT=12
 
 DESCRIPTION="Intel SPMD Program Compiler"
 HOMEPAGE="https://ispc.github.com/"
```
Comment (telans) [inline dev-lang/ispc/ispc-9999.ebuild:13]: 404?

File: dev-lang/ispc/ispc-9999.ebuild (hunk 1 above)
Comment (mgorny) [reply dev-lang/ispc/ispc-9999.ebuild:43]: Use `RelWithDebInfo`, that's the one that is least likely to break stuff.

File: dev-lang/ispc/ispc-9999.ebuild (hunk 2 above)
Comment (DarkDefender) [reply dev-lang/ispc/ispc-9999.ebuild]: Fixed.

**Review by juippis — COMMENTED** [review]
```
>>> Install dev-lang/ispc-9999 into /var/tmp/portage/dev-lang/ispc-9999/image
Traceback (most recent call last):
  File "/usr/lib/portage/python3.9/doins.py", line 594, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/usr/lib/portage/python3.9/doins.py", line 582, in main
    if _doins(
  File "/usr/lib/portage/python3.9/doins.py", line 434, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
  File "/usr/lib/portage/python3.9/doins.py", line 370, in install_file
    return self._ins_runner.run(source, dest_dir)
  File "/usr/lib/portage/python3.9/doins.py", line 179, in run
    sstat = os.stat(source)
FileNotFoundError: [Errno 2] No such file or directory: b'/var/tmp/portage/dev-lang/ispc-9999/work/ispc-9999_build/examples/*'
 * ERROR: dev-lang/ispc-9999::gentoo failed (install phase):
 *   doins failed
```
Could you fix this while at it?

Rest looks good to me.
(To clarify, installing this fails with 'examples' IUSE)

**Comment by DarkDefender** [comment]
> ```
> Could you fix this while at it?
> 
> Rest looks good to me.
> (To clarify, installing this fails with 'examples' IUSE)

Done :)

**Review by juippis — APPROVED** [review]
Thanks!

**Comment by kobboi** [comment]
Live ebuild fails

**Comment by kobboi** [comment]
* Applying ispc-9999-llvm.patch ...
patching file CMakeLists.txt
Hunk #1 succeeded at 275 (offset 57 lines).
Hunk #2 succeeded at 485 with fuzz 1 (offset 83 lines).
patching file src/llvmutil.cpp
Hunk #1 succeeded at 47 (offset 5 lines).
 [ ok ]
 * Applying ispc-9999-werror.patch ...
patching file CMakeLists.txt
Hunk #1 FAILED at 352.
1 out of 1 hunk FAILED -- saving rejects to file CMakeLists.txt.rej
